### PR TITLE
chore(post): show args passed

### PR DESCRIPTION
**Pull request by georgettica**
this is good in case you accidentally provide too many args by mistake

### CODE REVIEW

**Comment by georgettica** [comment]
related to [OSD-9903](https://issues.redhat.com/browse/OSD-9903)

**Comment by georgettica** [comment]
related to OSD-9903

**Comment by shibumi** [comment]
@georgettica  would it make sense to only enable this when a `--verbose` flag is being set?

**Comment by shibumi** [comment]
@georgettica  Also: I don't see where your code is related to the Issue you are working on. Can you add a comment or an explanation how your code solves the issue described in: https://issues.redhat.com/browse/OSD-9903 ?

**Comment by georgettica** [comment]
Seems ok to me, but I am already printing the command sent.
Once we move it to verbose, no on will use them

File: cmd/servicelog/list.go [hunk 1]
```diff
@@ -34,6 +34,10 @@ var listCmd = &cobra.Command{
 			}
 		}()
 
+		if len(args) != 1 {
+			log.Infof("The amount of arguments passed are: %d and they are '%v'", len(args), args)
```
Comment (fahlmant) [inline cmd/servicelog/list.go:38]:
```suggestion
			log.Infof("Too many arguments. Expected 1m got %d", len(args))
```

File: cmd/servicelog/post.go [hunk 2]
```diff
@@ -56,6 +56,9 @@ var postCmd = &cobra.Command{
 		}
 
 		var queries []string
+		if len(args) != 1 {
+			log.Infof("The amount of arguments passed are: %d and they are '%v'", len(args), args)
```
Comment (fahlmant) [inline cmd/servicelog/post.go:60]:
```suggestion
			log.Infof("Too many arguments. Expected 1m got %d", len(args))
```

File: cmd/servicelog/list.go (hunk 1 above)
Comment (shibumi) [reply cmd/servicelog/list.go:38]: Ah this clarifies my question I had!

Comment (shibumi) [reply cmd/servicelog/list.go:38]: @fahlmant  shouldn't it be:"Too many arguments. Expected 1, but got %d"? Is the `1m` a typo?

Comment (fahlmant) [reply cmd/servicelog/list.go:38]: yep, 1m is a typo. Fixed. And imo we should exit

File: cmd/servicelog/list.go [hunk 3]
```diff
@@ -34,6 +34,10 @@ var listCmd = &cobra.Command{
 			}
 		}()
 
+		if len(args) != 1 {
+			log.Infof("Too many arguments. Expected 1 got %d", len(args))
```
Comment (shibumi) [inline cmd/servicelog/list.go:38]: @georgettica  do we want to exit when the args don't match or is this not necessary?

Comment (georgettica) [reply cmd/servicelog/list.go:38]: No, I accepted the comment but meant for this to be informational, in case we hit this later on

**Comment by shibumi** [comment]
Oh! I didn't know that approve would "automerge" this. Sorry @fahlmant

**Comment by georgettica** [comment]
Yeah, this is the automation we should have had in CAD, not there yet

**Pull request merged**